barcodes_generator_product/models/product_template.py

```diff
@@ -11,7 +11,7 @@ class ProductTemplate(models.Model):
 
     # Related to display product product information if is_product_variant
     barcode_rule_id = fields.Many2one(
-        related="product_variant_ids.barcode_rule_id",
+        # related="product_variant_ids.barcode_rule_id",
```

Comment:
I dont think that removing thé related is a good idea. Could you elaborate the current use case that is failing ?